github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -71,6 +71,29 @@ periodics:
       image: quay.io/kubevirtci/flakefinder:v20220505-8aeca1a5
       name: ""
       resources: {}
+- annotations:
+    testgrid-create-test-group: "false"
+  cluster: ibm-prow-jobs
+  cron: 25 1 * * *
+  decorate: true
+  extra_refs:
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+    work_dir: true
+  name: periodic-push-kubevirt-flakefinder-report-values
+  spec:
+    containers:
+    - args:
+      - |
+        go run robots/cmd/push-flakefinder-results/main.go --pushgateway-url=http://pushgateway
+      command: [ "/usr/local/bin/runner.sh", "/bin/bash", "-ce" ]
+      env:
+      - name: GIMME_GO_VERSION
+        value: "1.16"
```

Comment:
Should we try and use a more recent version of go here?

Reply:
go.mod points to 1.16 still :(

Reply:
But good point, I think we should update the go version since it's pretty old. But we need presubmits for releng dir. I'll look into this.

Reply:
https://github.com/kubevirt/project-infra/pull/2357

Reply:
But maybe not in this PR.